src/txadmission.cpp

```diff
@@ -703,11 +703,6 @@ bool ParallelAcceptToMemoryPool(Snapshot &ss,
     {
         featureFlags |= SCRIPT_ENABLE_CHECKDATASIG;
     }
-    if (IsNov2019Activated(chainparams.GetConsensus(), chainActive.Tip()))
-    {
-        featureFlags |= SCRIPT_ENABLE_SCHNORR_MULTISIG;
-        featureFlags |= SCRIPT_VERIFY_MINIMALDATA;
```

Comment:
I don't see where SCRIPT_ENABLE_SCHNORR_MULTISIG is added to any of either the Standard or Manadatory flags (in policy.h or script/standard.h), SCRIPT_VERIFY_MINIMALDATA however is defined in the Standard flags...also, in the lines above you could probably take out "featureFlags |= SCRIPT_ENABLE_CHECKDATASIG;"  since it's already defined in Standard flags?

Reply:
both are added to the set of mandatory script in `standard.h`

see https://github.com/BitcoinUnlimited/BitcoinUnlimited/pull/2104/files#diff-117f0ad2885463dcf2b4cf1a5038939eR45

I should have removed `SCRIPT_VERIFY_MINIMALDATA` from policy.h 

WRT `SCRIPT_ENABLE_CHECKDATASIG` you are probably correct but I have to look more carefully.

Reply:
You're right, I missed that..

Reply:
you were right, I removed the `featureFlags |= SCRIPT_ENABLE_CHECKDATASIG` just left a stub for future features flags addition